stable/cluster-autoscaler/values.yaml

```diff
@@ -11,7 +11,7 @@ cloudProvider: aws
 
 image:
   repository: gcr.io/google_containers/cluster-autoscaler
-  tag: v0.6.0
+  tag: v1.1.0
```

Comment:
v1.1 is for Kubernetes 1.9, according to the docs. The v1.0 is for k8s 1.8 and v0.6.x (latest being v0.6.2, last time I checked) is for k8s 1.7. From what I gather, I don't think we want to upgrade to a version for a k8s that's still in beta?